comm.c

```diff
@@ -312,9 +318,10 @@ void str2ansi( char *p2, char *p1, int start, int stop )
     MYSQL_ROW mysqlRow;
 
     char *mysqlServer = "localhost";
-    char *mysqlUser = "root";
-   char *mysqlPassword = "kakka"; /* set me first */
```

Comment:
Questa è una password di default, poi c'e' modo di fare l'override da configurazione vero?

Reply:
Hai commentato il codice vecchio.... quello nuovo è verde e a destra :D

Reply:
e contiene variabili definite in config.h